apps/_components/src/ColorModal/ColorModal.tsx

```diff
@@ -52,7 +52,7 @@ export const ColorModal = ({
   weight,
 }: ColorModalProps) => {
   return (
-    <Modal.Context>
+    <Modal.TriggerContext>
```

Comment:
So here it looks like someone did the mistake I was afraid people did. This `TriggerContext` is not needed her because `Modal.Trigger` is not used.

Reply:
This was a bad refactor from use when we changed the modal API. I don't think this is usually a big issue.

Reply:
Do you want me to resolve this for us?